_articles/fr/starting-a-project.md

```diff
@@ -103,71 +103,71 @@ Si vous avez besoin d'un budget ou d'un personnel dédié pour la promotion, les
 
 Si votre projet est sur GitHub, placer ces fichiers dans votre répertoire racine avec les noms de fichiers recommandés aidera GitHub à les reconnaître et à les faire apparaître automatiquement à vos lecteurs.
 
 ### Choisir une licence
 
-Une licence open source garantit que d'autres utilisateurs peuvent utiliser, copier, modifier et contribuer à votre projet sans aucune répercussion. Il vous protège également contre les situations juridiques collantes. **Vous devez inclure une licence lorsque vous lancez un projet open source.**
+Une licence open source garantit que d'autres utilisateurs peuvent utiliser, copier, modifier et contribuer à votre projet sans aucune répercussion. Il vous protège également contre les situations juridiques épineuses. **Vous devez inclure une licence lorsque vous lancez un projet open source.**
 
 Le travail juridique n'est pas amusant. La bonne nouvelle est que vous pouvez copier et coller une licence existante dans votre dépôt. Cela ne prendra qu'une minute pour protéger votre dur labeur.
 
-[MIT](https://choosealicense.com/licenses/mit/), [Apache 2.0](https://choosealicense.com/licenses/apache-2.0/), et [GPLv3](https://choosealicense.com/licenses/gpl-3.0/) sont les licences Open Source les plus populaires, mais [il existe d'autres options](https://choosealicense.com) à choisir.
+[MIT](https://choosealicense.com/licenses/mit/), [Apache 2.0](https://choosealicense.com/licenses/apache-2.0/), et [GPLv3](https://choosealicense.com/licenses/gpl-3.0/) sont les licences Open Source les plus populaires, mais vous pouvez choisir [d'autres options](https://choosealicense.com).
 
 Lorsque vous créez un nouveau projet sur GitHub, vous avez la possibilité de sélectionner une licence. L'inclusion d'une licence open source rendra votre projet GitHub open source.
 
 ![Choisissez une licence](/assets/images/starting-a-project/repository-license-picker.png)
 
-Si vous avez d'autres questions ou préoccupations concernant les aspects juridiques de la gestion d'un projet open source, [nous vous avons couvert](../legal/).
+Si vous avez d'autres questions ou préoccupations concernant les aspects juridiques de la gestion d'un projet open source, [nous avons ce qu'il vous faut](../legal/).
 
 ### Ecrire un fichier README
 
 Les fichiers README font plus qu'expliquer comment utiliser votre projet. Ils expliquent également pourquoi votre projet est important et ce que vos utilisateurs peuvent en faire.
 
-Dans votre fichier README, essayez de répondre aux questions suivantes:
+Dans votre fichier README, essayez de répondre aux questions suivantes :
 
-* Que fait ce projet?
-* Pourquoi ce projet est-il utile?
-* Comment puis-je commencer?
-* Où puis-je obtenir plus d'aide, si j'en ai besoin?
+* Que fait ce projet ?
```

Comment:
Please avoid adding extra space here and elsewhere.

Reply:
Using the French punctuation rules, you should put a space before exclamation marks, question marks, colons, etc.
[Here](https://www.iwillteachyoualanguage.com/learn/french/french-tips/french-punctuation)'s a quick guide explaining this rule of French grammar.